api/controllers/v1/jobs.ts

```diff
@@ -23,12 +23,14 @@ export const TriggerLocalBuild = async (event: any = {}, context: any = {}): Pro
     consoleLogger.info(body.jobId, 'enqueuing Job');
     await sqs.sendMessage(new JobQueueMessage(body.jobId, JobStatus.inQueue), c.get('jobUpdatesQueueUrl'), 0);
     consoleLogger.info(body.jobId, 'Job Queued Job');
+    client.close();
```

Comment:
Just curious if the uses of `client.close()` should be awaited, or if it's okay to keep as promise? Wasn't sure if the promise would be resolved before process exited/lambdas returned (or however lambdas finish)

Reply:
Heh - we were thinking the same thing at the same time. Just pushed a commit that adds awaits!

Reply:
Nice timing 🤠